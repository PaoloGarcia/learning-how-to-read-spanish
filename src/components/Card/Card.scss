.Card {
   margin: 6px;

   button.ant-btn-round.ant-btn-sm {
      height: 60px;
      width: 60px;

      span {
         font-size: 24px;
      }
   }
}