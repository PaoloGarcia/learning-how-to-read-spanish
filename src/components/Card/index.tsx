import { Button } from "antd"
import "./Card.scss"

type Props = {
    text: string,
    sound?: string
}

function Card({ text, sound }: Props): JSX.Element {
    return (
        <div className="Card">
            <Button type="primary" shape="round" size="small" ghost>
                {text}
            </Button>
        </div>
    )
}

export default Card
