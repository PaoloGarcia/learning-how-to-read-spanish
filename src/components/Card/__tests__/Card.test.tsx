import React from "react"
import { render } from "@testing-library/react"
import Card from ".."

describe("Snapshot testing", () => {
    it("renders component without args", () => {
        const { asFragment } = render(<Card text="P" />)
        const firstRender: DocumentFragment = asFragment()
        expect(firstRender).toMatchSnapshot()
    })
})
