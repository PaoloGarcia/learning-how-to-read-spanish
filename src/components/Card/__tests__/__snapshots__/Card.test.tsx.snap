// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing renders component without args 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <button
      class="ant-btn ant-btn-primary ant-btn-round ant-btn-sm ant-btn-background-ghost"
      type="button"
    >
      <span>
        P
      </span>
    </button>
  </div>
</DocumentFragment>
`;
